src/options.rs

```diff
@@ -57,18 +57,26 @@ impl FromStr for Sanitizer {
 
 #[derive(Clone, Debug, StructOpt)]
 pub struct BuildOptions {
-    #[structopt(short = "O", long = "release")]
+    #[structopt(short = "D", long = "debug", conflicts_with = "release")]
+    /// Build artifacts in debug mode, without optimizations
+    pub debug: bool,
```

Comment:
Let's name this `--dev` to match the cargo profile name, and to avoid confusion like "how is this different from `--debug-assertions`?"

Reply:
Done